app/controllers/api/teams.php

```diff
@@ -90,6 +92,10 @@
             }
         }
 
+        if (!empty($user->getId())) {
```

Comment:
What is the cause behind this addition?

Reply:
If a user is creating a team, I need his user id to re-fetch his roles 👍🏻